src/read_csv.py

```diff
@@ -56,23 +62,45 @@ def main():
     """Run "python3 read_csv.py transform <nameOfFile>" to read in the CSV and transform it into a better format
     If no commandLine arguments are passed into it, then it passed the CSV into pandas."""
     if len(sys.argv) > 1:
-
         # CASE: transform the CSV from the bad input to the better one
         if sys.argv[1] == "transform":
-            file_name = sys.argv[2]
-            better_file_name = "better" + file_name
-            with open(file_name, 'r') as f:
-                reader = csv.reader(f)
-                data_list = list(reader)
-            transform_csv_from_bad_format_to_better_format(data_list, better_file_name)
+            directory_path = PATH + "/../data/csv_files/"
+            file = sys.argv[2]
+            transform_file(directory_path, file)
         else:
             file_name = sys.argv[1]
             get_dataframe_from_csv(file_name)
 
-    # CASE: everything else we are trying to do.
+    # CASE: transforms every .csv file in csv_files to a better format
     else:
-        # No Sys args, just use test file
-        get_dataframe_from_csv("../data/csv_files/betterTestCSV.csv_files")  # Need to run transform first to get this file
+        transform_all_files()
+
+def transform_all_files():
+    '''
+    Loops over every .csv file and calls transform_file function on it
+    :return: None (Output better files in data/better_csv_files)
+    '''
+    directory_path = PATH + "/../data/csv_files/"
```

Comment:
Same comment as below on parameterizing this

Reply:
I think having this directory_path as a parameter is a good call